db/schema.rb

```diff
@@ -25,18 +42,11 @@
     t.datetime "updated_at", null: false
     t.index ["style_id"], name: "index_recipes_on_style_id"
     t.index ["user_id"], name: "index_recipes_on_user_id"
-=======
-ActiveRecord::Schema.define(version: 20180310134810) do
+  end
 
-  create_table "ingredients", force: :cascade do |t|
+  create_table "styles", force: :cascade do |t|
     t.string "name"
-    t.string "category"
```

Comment:
you've changed the styles table in the schema, but there is no associated migration committed in this PR. Please either fix the schema or add the migration